core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Config.java

```diff
@@ -14,26 +14,29 @@
 package io.trino.server.security.oauth2;
 
 import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
 import io.airlift.configuration.Config;
 import io.airlift.configuration.ConfigDescription;
 import io.airlift.configuration.ConfigSecuritySensitive;
 import io.airlift.configuration.DefunctConfig;
+import io.airlift.configuration.LegacyConfig;
 import io.airlift.configuration.validation.FileExists;
 import io.airlift.units.Duration;
 import io.airlift.units.MinDuration;
 
 import javax.validation.constraints.NotNull;
 
 import java.io.File;
+import java.util.Collections;
```

Comment:
This commit is missing the doc changes.   I'd reorder this change so it aligns with the removal in the previous commit, and then I'd squash it in  to that commit

Reply:
👍